theme/themes/eea/views/item.variables

```diff
@@ -57,12 +57,13 @@
 @imageBorder: none;
 
 /* Image & Icon size */
-@tinyMediaSize   : 1.5rem;
-@smallMediaSize  : 3rem;
-@mediumMediaSize : 4rem;
-@largeMediaSize  : 8rem;
-@bigMediaSize    : 12.5rem;
-@hugeMediaSize   : 25rem;
+@miniMediaSize   : 1.5rem;
+@tinyMediaSize   : 2rem;
+@smallMediaSize  : 2.5rem;
+@mediumMediaSize : 3rem;
+@largeMediaSize  : 3.5rem;
+@bigMediaSize    : 4rem;
+@hugeMediaSize   : 4.5rem;
 
```

Comment:
@ichim-david  Here are the new sizes for the item media. We should check if we need any updates. 
E.g. the mini (24px) looks very small and you'll probably never use it.